cypress/components/grid/grid.cy.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jest/valid-expect, jest/valid-expect-in-promise */
```

Comment:
Can we get rid of this disable line?

Reply:
This was added at James's suggestion as the 'expect' which requires it is difficult to replace - when the eslint rules are redefined it shouldn't be required anyway.

Reply:
Alright, that makes sense. Alternatively I think we could use ".should("be.gte", number1). and("be.lte", number2)" to avoid 'expect'

Reply:
actually we, can use `cy.wrap($element).should(...)` instead of except. Or use an `alias`

Reply:
I had tried numerous combinations of your suggestions yesterday and was getting the same "Promise should be returned to test its fulfillment or rejection" error using cy.wrap. Anyway, what I've pushed a change which seems to be working without the disable line.